deployment/hasura/metadata/actions.graphql

```diff
@@ -191,7 +191,7 @@ type MerlinSimulationResponse {
 }
 
 type ValidationResponse {
-  errors: [ValidationNotice!]
+  errors: ValidationNotice!
   success: Boolean!
```

Comment:
Change was not properly reverted.

The original code was `errors: [ValidationNotice!]`, which means that `errors` is optional, but if it's there, there are no `null` entries. 

`errors: [ValidationNotice]!` means that `errors` _must_ be present, but that it may have `null` entries.